pkg/sqlite/file.go

```diff
@@ -575,6 +575,23 @@ func (qb *FileStore) find(ctx context.Context, id file.ID) (file.File, error) {
 
 // FindByPath returns the first file that matches the given path. Wildcard characters are supported.
 func (qb *FileStore) FindByPath(ctx context.Context, p string) (file.File, error) {
+
+	ret, err := qb.FindAllByPath(ctx, p)
+
+	if err != nil {
+		return nil, err
+	}
```

Comment:
the existing FindByPath ignored sql.ErrNoRows as well ?
```golang
if err != nil && !errors.Is(err, sql.ErrNoRows) {
               return nil, fmt.Errorf("getting file by path %s: %w", p, err)
        }
```

Reply:
Yes, that was the bug, I've validated that it now works correctly.